### ISSUE TITLE
Feat Req for PL manager:  Add a delete all fully watched vids, leaving partially watched undeleted

### ISSUE THREAD

**Issue by jsburwell** [NabokD/pockettube#118]
Add a delete all fully watched vids, leaving partially watched undeleted.  Perhaps allow user to specify what qualifies as "fully watched", 50%, 90%, etc.

Sometimes you just want to clear the fully watched videos from the top for your Watch Later without deleting that last video that you only watched part of and want to continue...

**Comment by NabokD** [comment]
Thank you for sharing!

**Comment by Th1rt3nth** [comment]
This feature is a must-have when you're watching videos on different devices or starting a new YouTube session each time.

In my opinion, there's no need for additional qualifiers—the "watched" filter should delete only **fully** watched videos. If there are partially watched videos that you don’t want to finish, you just mark them as watched.

I’m wondering why not many people seem to struggle with this issue. Do you guys always watch videos until the end? Or how do you prevent fully unwatched videos from getting lost?

**Comment by Th1rt3nth** [comment]
By the way, I was referring to hiding watched videos in the Deck. I just realized that this issue might be about deleting videos somewhere else. If that's the case, sorry for the misunderstanding.

**Comment by NabokD** [comment]
Yes, it'll be included in the next major release in March-April. It'll be the universal setting for all views incl. Deck.

**Comment by NabokD** [comment]
Hello,
The new version 16.6.0 is available with watched threshold feature, you can see it in action on the video
https://www.youtube.com/watch?v=sDSpl798w9c&t=50s
